src/main/resources/db/changelog/changes/01-create-books-table.yaml

```diff
@@ -0,0 +1,49 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-book-table
+      author: Nikolai
+      changes:
+        - createTable:
+            tableName: books
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: title
+                  type: varchar(255)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: author
+                  type: varchar(255)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: isbn
+                  type: varchar(255)
+                  constraints:
+                    nullable: false
+                    unique: true
+              - column:
+                  name: price
+                  type: decimal(19, 2)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: description
+                  type: text
+              - column:
+                  name: cover_image
+                  type: text
+              - column:
+                  name: is_deleted
+                  type: bool
+                  constraints:
+                    nullable: false
+        - sql:
+            sql: "ALTER TABLE books ALTER COLUMN is_deleted SET DEFAULT false"
```

Comment:
why do you need this sql? remove it

Reply:
I haven't found another way to set the "is-deleted" default column value to false